### PR TITLE
[av][android] Add support for react-native 0.76

### PR DESCRIPTION
**Pull request by gabrieldonadel**
# Why

Related to ENG-13531

# How

Update expo-av to support react-native 0.76 based on https://github.com/react-native-community/discussions-and-proposals/discussions/816

# Test Plan

Run BareExpo using react-native 0.75 and 0.76

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: packages/expo-av/android/CMakeLists.txt [hunk 1]
```diff
@@ -43,6 +43,16 @@ target_link_libraries(
         ${LOG_LIB}
         fbjni::fbjni
         ReactAndroid::jsi
-        ReactAndroid::reactnativejni
         android
 )
+if (ReactAndroid_VERSION_MINOR GREATER_EQUAL 76)
```
Comment (Kudo) [inline packages/expo-av/android/CMakeLists.txt:48]: i think `ReactAndroid_VERSION_MINOR` is undefined?

**Review by Kudo — CHANGES_REQUESTED** [review]
looks good but the `ReactAndroid_VERSION_MINOR` is undefined

File: packages/expo-av/android/CMakeLists.txt (hunk 1 above)
Comment (gabrieldonadel) [reply packages/expo-av/android/CMakeLists.txt:48]: Not really, in 0.76 this is what I get when logging `ReactAndroid_VERSION_MINOR is defined: ${ReactAndroid_VERSION_MINOR}"`

image

Comment (Kudo) [reply packages/expo-av/android/CMakeLists.txt:48]: oh i see. it is from cmake: https://cmake.org/cmake/help/latest/variable/PROJECT-NAME_VERSION_MINOR.html#variable:%3CPROJECT-NAME%3E_VERSION_MINOR.
sorry for this false alarm

**Review by Kudo — APPROVED** [review]

**Pull request merged**